assets/src/styles/components/_comments.scss

```diff
@@ -0,0 +1,132 @@
+.comment-list {
+	list-style: none;
+	margin: 0;
+	padding: 0;
+}
+
+.comment-list .comment {
+
+	margin: 0;
+	padding: $base-line-height auto $base-line-height;
+
+	@media #{ $mq-lg-up } {
+		padding-top: $base-line-height * 2;
+		padding-bottom: $base-line-height * 2;
+	}
+
+	& + .comment {
+		border-top: 1px solid $border-color;
+		margin-top: 0;
+		padding-top: $base-line-height;
+
+		@media #{ $mq-lg-up } {
+			padding-top: $base-line-height * 2;
+		}
+
+	}
+
+	.children {
+		list-style: none;
+		margin: 0 0 0 $gutter-width;
+		padding: 0;
+
+		.comment {
+			padding-left: $gutter-width;
+		}
+
+	}
+}
+
+
+.comment-meta {
+
+	margin-bottom: $base-line-height;
+	padding-left: calc( 3rem + #{ $gutter-width } );
+
+	.avatar	 {
+		width: 3rem;
+		height: 3rem;
+		float: left;
+		margin-left: calc( -3rem - #{ $gutter-width } );
+		border-radius: 2px;
+	}
+
+	.comment-author {
+		@include font-heading;
+		color: $hm-dark-grey;
+
+		a {
+			color: inherit;
+		}
+	}
+
+	.comment-metadata {
+		@include text-sm;
+
+		a {
+			color: inherit;
+		}
+	}
+
+	.edit-link {
+		display: none;
+	}
+}
+
+.comment-reply-link {
+	@extend .Btn;
+	@extend .Btn-Small;
+	@extend .Btn-Tertiary;
+}
+
+.comment-reply-title a {
+	@extend .Btn;
+	@extend .Btn-Small;
+	float: right;
+}
+
+.comment-awaiting-moderation {
+	background: $hm-light-grey;
+	border-radius: 2px;
+	padding: $base-line-height / 2 $gutter-width;
+	margin-left: calc( -3rem - #{ $gutter-width } );
```

Comment:
Assuming the 3rem here is from the avatar width? Might be worth making the avatar size a variable. 
1. Easier to read and understand when used away from other avatar declarations (like this)
2. If we decide to change the size, can be done in one place
3. Can also be overridden by a child theme